### PR TITLE
Merge `next` into `main`

### PR DESCRIPTION
**Pull request by Eomm**

### CODE REVIEW

**Review by gurgunday — APPROVED** [review]
lgtm

**Comment by gurgunday** [comment]
I created the v3 branch

File: package.json [hunk 1]
```diff
@@ -36,19 +36,19 @@
   "devDependencies": {
     "ajv-cli": "^5.0.0",
     "ajv-errors": "^3.0.0",
-    "ajv-i18n": "^4.0.1",
+    "ajv-i18n": "^4.2.0",
     "ajv-merge-patch": "^5.0.1",
     "cronometro": "^3.0.1",
-    "fastify": "^4.0.0",
+    "fastify": "^4.26.2",
```
Comment (jsumners) [inline package.json:42]:
```suggestion
    "fastify": "^5.0.0-alpha.2",
```

File: package.json [hunk 2]
```diff
@@ -36,19 +36,19 @@
...
+    "tap": "^19.0.0",
     "tsd": "^0.31.0"
   },
   "dependencies": {
-    "ajv": "^8.11.0",
-    "ajv-formats": "^2.1.1",
-    "fast-uri": "^2.0.0"
+    "ajv": "^8.12.0",
+    "ajv-formats": "^3.0.1",
+    "fast-uri": "^2.3.0"
```
Comment (jsumners) [inline package.json:52]:
```suggestion
    "fast-uri": "^3.0.0"
```

**Review by gurgunday — APPROVED** [review]

**Review by jsumners — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**